process_manager/views/partials.py

```diff
@@ -0,0 +1,54 @@
+"""View functions for partials."""
+
+import django_tables2
+from django.contrib.auth.decorators import login_required
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from druncschema.process_manager_pb2 import (
+    ProcessInstance,
+)
+
+from ..process_manager_interface import get_session_info
+from ..tables import ProcessTable
+
+
+@login_required
+def process_table(request: HttpRequest) -> HttpResponse:
+    """Renders the process table.
+
+    This view may be called using either GET or POST methods. GET renders the table with
+    no check boxes selected. POST renders the table with checked boxes for any table row
+    with a uuid provided in the select key of the request data.
+    """
+    selected_rows = request.POST.getlist("select", [])
```

Comment:
When it is a GET method, does POST still exist with no `select`? Otherwise, how do we distinguish between a GET and a POST?

Reply:
It does still exist. We make use of the case that there is no difference in behaviour of the function when there is a GET request or a POST request with no data.